contents/docs/cdp/batch-exports/index.md

```diff
@@ -32,12 +33,78 @@ A batch export is executed in batch runs depending on the configured frequency.
 
 As an example, creating a batch export of events with daily frequency today will schedule the first batch run to start right as tomorrow begins. Thus, the data exported are events that PostHog received from 00:00:00 until 23:59:59 of today.
 
-Batch runs are configured on the [exports page](https://app.posthog.com/exports) in your PostHog instance.
-
 > **Note:** When deciding which data falls within the bounds of a specific batch run, we look at the time when the data landed in PostHog's ClickHouse database. This means that network and processing delays can make an event that was sent within the bounds of a batch run fall in a future run.
 
-## How does this differ from the old export system?
+### Tracking progress
+
+On each batch export view we will be presented with a list of the latest runs executed:
+
+![batch export runs](../../../images/docs/batch-exports/batch-exports-runs.png)
+
+Each run has an indicator of the state (which can be either "Starting", "Running", "Failed", or "Completed"), the boundaries for the data exported (data interval start and data interval end), when the run actually started, and the option of retrying a specific run.
+
+
+## Historical exports
+
+Batch exports can export data that landed in PostHog before the batch export was created, known as historical data. For this, no new batch export has to be created: A batch export already knows the destination where we wish to export historical data, it only requires the boundaries for the historical export, in other words a start and an end date.
```

Comment:
```suggestion
You can use batch exports for past data stored in PostHog, known as historical data. For this, you don't need to create a new batch export. The batch export already knows the destination where we wish to send historical data. It only requires the boundaries for the data you want to export, in other words, a start and an end date.
```
This is worded a little confusingly.

Reply:
Happy to take the suggestion for rewording 👍 Thank you